### PR TITLE
Add visions

### PR DESCRIPTION
**Pull request by philip-khor**
`visions` is a dependency for newer versions of `pandas-profiling` 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge/help-python this recipe is ready for review/merge, thanks!

File: recipes/visions/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+requirements:
+  host:
+    - attrs
+    - networkx
+    - numpy >=1.14.6
+    - pandas >=0.25.3
+    - pip
+    - python >=3.5
+    - tangled-up-in-unicode >=0.0.3
```
Comment (scopatz) [inline recipes/visions/meta.yaml]: Are all these needed? I think python and pip should be enough

File: recipes/visions/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    - mypy
+    - pytest >=5.2.0
+    - pytest-black
+    - pytest-mypy
+    - pytest-ordering
+    - pytest-rerunfailures
+    - pytest-runner
+    - pytest-sugar
+    - pytest-tldr
+    - imagehash
```
Comment (scopatz) [inline recipes/visions/meta.yaml]: It doesn't seem as though you are actually running pytest. I am not sure you need these

File: recipes/visions/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+about:
+  home: https://github.com/dylan-profiler/visions
+  license: BSD 4-Clause
```
Comment (scopatz) [inline recipes/visions/meta.yaml]: I think the SPDX identifier is `BSD-4-Clause`

**Pull request merged**